ompi/mca/mtl/ofi/mtl_ofi.h

```diff
@@ -999,6 +999,67 @@ ompi_mtl_ofi_send_generic(struct mca_mtl_base_module_t *mtl,
     return ofi_req.status.MPI_ERROR;
 }
 
+__opal_attribute_always_inline__ static inline int
```

Comment:
This function shouldn't be always-inlined.  Yes, I know someone got this entire component wrong before, but we don't need to propogate the badness.